addons/l10n_ch/wizard/qr_invoice_wizard.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api, _
+from odoo.exceptions import UserError
+
+
+class QrInvoiceWizard(models.TransientModel):
+    '''
+    Wizard :
+    When multiple invoices are selected to be printed in the QR-Iban format,
+    this wizard will appear if one or more invoice(s) could not be QR-printed (wrong format...)
+    The user will then be able to print the invoices (in the format available, priority : QR --> ISR --> normal)
+     or to see a list of the non-QR/ISR invoices.
+    The non-QR/ISR invoices will have a note logged in their chatter, detailing the reason of the failure.
+    '''
+    _name = 'l10n_ch.qr_invoice.wizard'
+    _description = 'Handles problems occurring while creating multiple QR-invoices at once'
+
+    nb_qr_inv = fields.Integer(default=lambda self: len(self._context.get('default_qr_inv_ids', [])))
+    nb_isr_inv = fields.Integer(default=lambda self: len(self._context.get('default_isr_inv_ids', [])))
+    nb_classic_inv = fields.Integer(default=lambda self: len(self._context.get('default_classic_inv_ids', [])))
```

Comment:
You are depending too much from the context. A better design will be to:
- populate a m2m fields pointing to `account.move` base on `active_ids` in `default_get`.
- make all `nb...` & `...text` fields computed depending of this newly field